python/pydantic_core/_pydantic_core.pyi

```diff
@@ -884,14 +1001,36 @@ def list_all_errors() -> list[ErrorTypeInfo]:
     """
 @final
 class TzInfo(datetime.tzinfo):
+    """A `pydantic-core` implementation of the `datetime.tzinfo` class.
+
+    Docstrings for attributes sourced from the base class, [`datetime.tzinfo`](https://docs.python.org/3/library/datetime.html#datetime.tzinfo)."""
+
     def tzname(self, _dt: datetime.datetime | None) -> str | None: ...
```

Comment:
```suggestion
    def tzname(self, _dt: datetime.datetime | None, /) -> str | None: ...
```
And for the others, let's follow the [definitions in typeshed](https://github.com/python/typeshed/blob/f58dac1d62d33bb2255b762783d06463c40f5065/stdlib/datetime.pyi#L15-L22).

Reply:
Good call, thanks for the reference!